models/Position.js

```diff
@@ -12,6 +12,14 @@ const Position = sequelize.define('position', {
             isInt: { msg: 'Event ID should be a number.' }
         },
     },
+    deleted: {
+        type: Sequelize.BOOLEAN,
+        allowNull: false,
+        defaultValue: '0',
```

Comment:
`false` as well

Reply:
It is working however, on a low level a boolean is just a bit anyway. But will change for clarity